### PR TITLE
fix: remove NA string replacement, fixed upstream

### PR DESCRIPTION
**Pull request by lparsons**
Wrapper previously failed if the optional output `qc_metrics` was not supplied due to NA replacement code that was not in a conditional. The code is no longer required since NA strings are now output correctly in https://github.com/deeptools/deepTools/pull/1002

### QC

* [x] I confirm that:

For all wrappers added by this PR, 

* there is a test case which covers any introduced changes,
* `input:` and `output:` file paths in the resulting rule can be changed arbitrarily,
* either the wrapper can only use a single core, or the example rule contains a `threads: x` statement with `x` being a reasonable default,
* rule names in the test case are in [snake_case](https://en.wikipedia.org/wiki/Snake_case) and somehow tell what the rule is about or match the tools purpose or name (e.g., `map_reads` for a step that maps reads),
* all `environment.yaml` specifications follow [the respective best practices](https://stackoverflow.com/a/64594513/2352071),
* the `environment.yaml` pinning has been updated by running `snakedeploy pin-conda-envs environment.yaml` on a linux machine,
* wherever possible, command line arguments are inferred and set automatically (e.g. based on file extensions in `input:` or `output:`),
* all fields of the example rules in the `Snakefile`s and their entries are explained via comments (`input:`/`output:`/`params:` etc.),
* `stderr` and/or `stdout` are logged correctly (`log:`), depending on the wrapped tool,
* temporary files are either written to a unique hidden folder in the working directory, or (better) stored where the Python function `tempfile.gettempdir()` points to (see [here](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir); this also means that using any Python `tempfile` default behavior works),
* the `meta.yaml` contains a link to the documentation of the respective tool or command,
* `Snakefile`s pass the linting (`snakemake --lint`),
* `Snakefile`s are formatted with [snakefmt](https://github.com/snakemake/snakefmt),
* Python wrapper scripts are formatted with [black](https://black.readthedocs.io).
* Conda environments use a minimal amount of channels, in recommended ordering. E.g. for bioconda, use (conda-forge, bioconda, nodefaults, as conda-forge should have highest priority and defaults channels are usually not needed because most packages are in conda-forge nowadays).

### CODE REVIEW

File: bio/deeptools/plotfingerprint/test/Snakefile [hunk 1]
```diff
@@ -2,21 +2,20 @@ rule plot_fingerprint:
...
+        qc_metrics="plot_fingerprint/qc_metrics.txt",
     log:
-        "logs/deeptools/plot_fingerprint.log"
+        "logs/deeptools/plot_fingerprint.log",
     params:
         # optional parameters
-        "--numberOfSamples 200 "
-    threads:
-        8
+        "--numberOfSamples 200 ",
```
Comment (fgvieira) [inline bio/deeptools/plotfingerprint/test/Snakefile:18]: For the sake of consistency, can you use `params.extra` instead?

```suggestion
        extra="--numberOfSamples 200",
```

Comment (fgvieira) [inline bio/deeptools/plotfingerprint/test/Snakefile:18]:
```suggestion
        extra="--numberOfSamples 200",
```

**Comment by fgvieira** [comment]
Looks good!
Wanted to make a suggestion, and ended up committing. Can you use `params.extra` instead?

**Comment by lparsons** [comment]
> Looks good! Sorry, but wanted to make a suggestion and ended up committing. Can you use `params.extra` instead?

Sure, I can do that. I took a look at the other deeptools wrappers and tried to make them more consistent, so I updated plotheatmap and plotprofile as well.

**Review by fgvieira — APPROVED** [review]

**Pull request merged**